save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/index/CardUser.kt

```diff
@@ -81,6 +97,9 @@ val cardUser: FC<IndexViewProps> = FC { props ->
             div {
                 className = ClassName("col-9")
                 props.userInfo?.let {
+                    p {
+                        +"Registered since ${it.createDate} !"
```

Comment:
also I am not sure that create date format is right: need to show year month and date, without time

Reply:
Fixed